packages/react/src/React.ml

```diff
@@ -386,13 +390,14 @@ type element =
   | Consumer of element
   | Suspense of { key : string option; children : element; fallback : element }
 
-and client_props = (string * client_prop) list
+and client_props = (string * rsc_value) list
 
-and client_prop =
+and rsc_value =
```

Comment:
I would still call this client_prop, rsc_value doesn't add any meaning.

Reply:
But rsc_value is used for the server action return. `client_prop` is limited to a component. 
Could it be `client_value`?

Reply:
Check for react name for this kind or use client_value